### ISSUE TITLE
axis ignored

### ISSUE THREAD

**Issue by 1Map** [proj4js/proj4js#305]
I have a proj as follow (note the **+axis=enu**)

`+proj=tmerc +lat_0=0 +lon_0=27 +k=1 +x_0=0 +y_0=0 +axis=enu +ellps=WGS84 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs`

Now when I run:

`var ll = proj4("+proj=tmerc +lat_0=0 +lon_0=27 +k=1 +x_0=0 +y_0=0 +axis=enu +ellps=WGS84 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs", "EPSG:4326", [lastycoord, lastxcoord]);
`
it gives me axactly the same result as when I use the proj4 without the **+axis**, example:

`var ll = proj4("+proj=tmerc +lat_0=0 +lon_0=27 +k=1 +x_0=0 +y_0=0 +ellps=WGS84 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs", "EPSG:4326", [lastycoord, lastxcoord]);
`
In other words, the +axis is ignored.

**Comment by Akuukis** [comment]
Bump, any progress on this?

**Comment by Akuukis** [comment]
@DaniEll-AT Oh, that makes sense. Thanks! 

I remember that no other axes worked (e.g. neu), but let me get a better example for such cases..

**Comment by fontsie** [comment]
I think this could be a good example:
```js
proj4("+proj=tmerc +lat_0=0 +lon_0=13.33333333333333 +k=1 +x_0=450000 +y_0=-5000000 +ellps=bessel +towgs84=577.326,90.129,463.919,5.137,1.474,5.297,2.4232 +units=m +axis=neu +no_defs ", "EPSG:4326",  [157571.79, 147114.52]);
// [9.530000047494815, 46.399999963719665]

proj4("+proj=tmerc +lat_0=0 +lon_0=13.33333333333333 +k=1 +x_0=450000 +y_0=-5000000 +ellps=bessel +towgs84=577.326,90.129,463.919,5.137,1.474,5.297,2.4232 +units=m +axis=enu +no_defs ", "EPSG:4326",  [157571.79, 147114.52]);
// [9.530000047494815, 46.399999963719665]
```

the +axis param is changed but the result is the same.
